src/syntax/ast.rs

```diff
@@ -82,6 +83,42 @@ pub struct StmtAssign {
     pub value: Option<ExprS>,
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct StmtFieldAssign {
+    pub access_modifier: Option<AccessModifier>,
+    pub field: Spanned<StmtAssign>,
+}
```

Comment:
Remove `Stmt` prefix? This is a data structure used for class declaration statements.